lib/mcb/commands/providers/edit.rb

```diff
@@ -0,0 +1,12 @@
+name 'show'
```

Comment:
I think this is meant to be `edit`?

Reply:
Pro tip: We don't actually need command names in these files, `Cri` takes it from the filename.